src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -687,11 +687,46 @@
                                             <tkcontrols:SettingsCard
                                                 MinHeight="0"
                                                 BorderThickness="0"
-                                                ContentAlignment="Right">
-                                                <TextBlock Opacity="{x:Bind DisabledOpacity}" Style="{ThemeResource SecondaryTextStyle}">
-                                                    <Run x:Uid="PowerLauncher_AuthoredBy" />
-                                                    <Run FontWeight="SemiBold" Text="{x:Bind Author}" />
-                                                </TextBlock>
+                                                ContentAlignment="Right"
+                                                IsEnabled="{x:Bind Enabled, Mode=OneWay}">
+                                                <StackPanel
+                                                    HorizontalAlignment="Right"
+                                                    Orientation="Horizontal"
+                                                    Spacing="5">
```

Comment:
```suggestion
                                                    Spacing="4">
```

Reply:
Any specific reason for this change? For me 4 feels to less and 6 feels to much. I like to keep 5.

(With 4 it looks more like a text character than a separator.)

Reply:
@niels9001
Regardless of my opinion, the spacing is updated.